site/installing/standalone.md

```diff
@@ -52,6 +52,22 @@ runs the Flux service and the Flux daemon in a single pod.
 kubectl create -f flux-deployment.yaml
 ```
 
+#### Note for Kubernetes >=1.6 with role-based access control (RBAC)
+
+You will need to provide fluxd with a service account which can access
+the namespaces you want to use Flux with. To do this, consult the
+example service account given in
+[flux-account.yaml](../../deploy/standalone/flux-account.yaml) (which
+puts essentially no constraints on the account) and the
+[RBAC documentation](https://kubernetes.io/docs/admin/authorization/rbac/),
+and create a service account in whichever namespace you put fluxd
+in. You may need to alter the `namespace: default` lines, if you adapt
+the example.
+
```

Comment:
You must provide `fluxd`  with a service account that can access
 the namespaces you want to use Flux with. Consult the
 example service account in  [flux-account.yaml](../../deploy/standalone/flux-account.yaml) (which has no constraints on the account) and the
 [RBAC documentation](https://kubernetes.io/docs/admin/authorization/rbac/),
 and then create a service account in the namespace that `fluxd` is running. Note that  the `namespace: default` lines may need to be altered, if you adapt
this example.